exasol_script_languages_container_tool/lib/utils/logging_redirection.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+
+class _Tee(object):
+    # Helper object which redirects calls to stdout/stderr to itself, prints to a give file and to real stdout/stderr
+    def __init__(self, name, mode):
+        self.file = open(name, mode)
+        self.stdout = sys.stdout
+        self.stderr = sys.stderr
+        sys.stdout = self
+        sys.stderr = self
+
+    def get_file(self):
+        return self.file
+
+    def __del__(self):
+        sys.stdout = self.stdout
+        sys.stderr = self.stderr
+        self.file.close()
+
+    def write(self, data):
+        self.file.write(data)
+        self.stdout.write(data)
+
+    def flush(self):
+        self.file.flush()
+
+
+def create_docker_build_creator(task_creator):
+    # Provides redirection of ALL logs to exaslct.log in the tasks job directory
+
+    # Parameters:
+    #   task_creator (function): creator function of the task
+    def create_docker_build():
+        task = task_creator()
+        tee = _Tee(f'{task.get_log_path()}/exaslct.log', "w")
```

Comment:
Do you have an example for the log file path. `task.get_log_path()` might be a subdirectory of the job-directory.

Reply:
Here is one
 .build_output/jobs/2021_05_31_11_01_19_DockerBuild/outputs/DockerBuild_1c5a98ee16/logs/exaslct.log

Reply:
thinking about, if we should show the whole path to exaslct or only the following part

Reply:
Ok, will keep as is then.